docs/site/angular-auth-oidc-client/docs/documentation/configuration.md

```diff
@@ -219,6 +219,24 @@ export const appConfig: ApplicationConfig = {
 bootstrapApplication(AppComponent, appConfig);
 ```
 
+The function `withAppInitializerAuthCheck` is provided to handle OAuth callbacks during app initialization
+phase. This replaces the need to manually call `OidcSecurityService.checkAuth(...)` or
+`OidcSecurityService.checkAuthMultiple(...)`.
+
+```ts
+...
+import { provideAuth, withAppInitializerAuthCheck } from 'angular-auth-oidc-client';
+
+...
+    provideAuth({
+      config: {
+        /* Your config here */
+      },
+    },
+    withAppInitializerAuthCheck()),
+...
+```
```

Comment:
I think it will be more clear if we provide a full example here.

```suggestion
```ts
import { ApplicationConfig } from '@angular/core';
import { provideAuth, withAppInitializerAuthCheck } from 'angular-auth-oidc-client';

export const appConfig: ApplicationConfig = {
  providers: [
    provideAuth({
      config: {
        /* Your config here */
      },
      withAppInitializerAuthCheck()
    }),
  ],
};

```
```

Reply:
Will do 👍